### ISSUE TITLE
Plugin doesn't handle web_accessible_resources' wildcards.

### ISSUE THREAD

**Issue by Bludater** [kevincharm/parcel-plugin-web-extension#12]
Thanks for your plugin!

Wildcards in `web_accessible_resources` throws error:
```
×  ...\src\manifest.json: Cannot resolve dependency './contentScripts\*' at ...\src\contentScripts\*'
    at Resolver.resolve (D:\InPATH\npm\node_modules\parcel\src\Resolver.js:69:17)
```

More info [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/web_accessible_resources#Wildcards) or [Chrome](https://developer.chrome.com/extensions/manifest/web_accessible_resources):
>`web_accessible_resources` entries can contain wildcards. For example, the following entry would also work to include the resource at "images/my-image.png":
>```JSON
>"web_accessible_resources": ["images/*.png"]
>```

**Comment by kevincharm** [comment]
Interesting. Should be a fairly straightforward fix by just globbing, which we already do with locales. I'll take a look next week or so, just about to move overseas so will be busy the next few days.

**Comment by kevincharm** [comment]
@Bludater please check out #16 and test it with your use case when you get a minute.

**Comment by kevincharm** [comment]
Merged #16, released in v1.4.0.
